lib/ts/recipe/accountlinking/recipe.ts

```diff
@@ -17,82 +17,835 @@ import error from "../../error";
 import { BaseRequest, BaseResponse } from "../../framework";
 import normalisedURLPath from "../../normalisedURLPath";
 import RecipeModule from "../../recipeModule";
-import SuperTokens from "../..";
```

Comment:
i have created a PR on top of this one with lots of small changes and bug fixes: https://github.com/supertokens/supertokens-node/pull/498/files